_wormtablemodule.c

```diff
@@ -22,16 +22,25 @@
 #include <db.h>
 #include "halffloat.h"
 
+/* $VR$: Dealing with Windows... */
+/* the declspec allows compiling of a dll in Visual Studio
```

Comment:
I think this comment would be more helpful if it was right above the ``__declspec`` define. This way, we know that its referring to exactly this thing.